src/ProgressOnderwijsUtils/XmlSerializableBase.cs

```diff
@@ -11,9 +11,6 @@ public static class XmlSerializerHelper
         => XmlSerializerHelper<T>.Deserialize(xml);
 
     public static string SerializeToString(object o)
-        => SerializeToString(o, null);
-
-    public static string SerializeToString(object o, XmlSerializerNamespaces? namespaces)
```

Comment:
Deze is op zich stukken minder gek dan die ToStringWithXmlDeclaration implementatie.

Als de api zo handig is, kunnen we ook overwegen puur de implementatie aan te passen, en de api te laten? Hangt een beetje af van waar we heen willen met de stap serialization.